src/wutil.cpp

```diff
@@ -862,6 +862,22 @@ static int compare(T a, T b) {
     return 0;
 }
 
+/// this is basically for builtin test
+bool file_id_t::compare_time(const file_id_t &rhs) const {
```

Comment:
I expect a "compare" method to return one of -1, 0 or 1.
If we return a bool, it should be called "less" or "is_older".